compositor_integration_tests/src/validation.rs

```diff
@@ -69,7 +69,7 @@ pub fn compare_audio_dumps(
     let expected_audio_packets = find_packets_for_payload_type(&expected_packets, 97);
     let actual_audio_packets = find_packets_for_payload_type(&actual_packets, 97);
 
-    let sample_rate = config().output_sample_rate;
+    let sample_rate = read_config().output_sample_rate;
```

Comment:
this should be read from original config, but I don't want to refactor that in this PR